### DISCUSSION TITLE
Hide Property Panel from Viewer

### DISCUSSION THREAD

**Discussion by candcconsulting** [iTwin/itwinjs-core#2704] [Q&A] [answered]
is there a property than can be passed to the react viewer to "hide" the property panel.
or does it have to be declared as a property panel and determine when it is displayed ?
Ie can it be turned off here
          <Viewer
            contextId={process.env.IMJS_CONTEXT_ID ?? ""}
            iModelId={process.env.IMJS_IMODEL_ID ?? ""}
            authConfig={{ oidcClient: AuthorizationClient.oidcClient }}
            onIModelConnected = {onIModelConnected}
            uiProviders = {[new SmartDeviceUiItemsProvider()]}
          />
or does it have to be declared as in the item sample showcase 

  useEffect(() => {
    if (iModelConnection) {
      PropertyFormattingApi.addSelectionListener(_onSelectionChanged);
    }
And create our a handler ?

**Comment by kmurari103** [comment, marked as answer]
You can pass **defaultUiConfig={{ hidePropertyGrid: true }}** in Viewer to hide the property panel , for other Viewer props please refer their readme [@itwin/web-viewer-react
](https://www.npmjs.com/package/@itwin/web-viewer-react)

<Viewer
contextId={process.env.IMJS_CONTEXT_ID ?? ""}
iModelId={process.env.IMJS_IMODEL_ID ?? ""}
authConfig={{ oidcClient: AuthorizationClient.oidcClient }}
onIModelConnected = {onIModelConnected}
uiProviders = {[new SmartDeviceUiItemsProvider()]}
**defaultUiConfig={{ hidePropertyGrid: true }}**
/>
